server/api/rotation.go

```diff
@@ -0,0 +1,254 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package api
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/mattermost/mattermost-plugin-solar-lottery/server/store"
+)
+
+type Rotation struct {
+	*store.Rotation
+
+	StartTime time.Time
+	Users     UserMap
```

Comment:
This confused me for a bit -- but I think it makes sense now: once a rotation has been "expanded", this usermap is filled-in (expanded) with each user's solar-lottery user struct, and their mattermost user struct. 
Probably should document for the next person to read the code.

Reply:
I am considering some of these interfaces/patterns to be still in flux. `Expand` should either become a normal template for all `stored`->`api` backfills, or should be replaced with a different template altogether. Inclined to punt on documenting until further matures.